packages/web/src/features/translation/TranslateWord.tsx

```diff
@@ -26,16 +26,16 @@ export default function TranslateWord({
   const { t } = useTranslation();
 
   return (
-    <li className="mx-2 mb-4 w-36">
+    <li className="mx-2 mb-4">
       <div
         id={`word-${word.id}`}
-        className={`font-serif mb-2 ${
-          originalLanguage === 'hebrew' ? 'text-2xl text-right' : 'text-lg'
+        className={`font-serif mb-2 text-center ${
```

Comment:
I'm not generally a fan of of center alignment. It is a general principle in UX design to align the edges of everything. But if the problem is that having a RTL and a LTR language on top of each other also looks weird, I would solve that problem by aligning everything to the right for Hebrew and to the left for Greek. That's what the current app does and I think it works well.

Reply:
I had never heard of that principle, but it makes sense. It looks better like you suggested.

Reply:
Aha! https://uxengineer.com/principles-of-design/alignment/